### PR TITLE
API version migration from v0.6 to v0.7

### PR DESCRIPTION
**Pull request by onthecrow**
## Description

- Library:
  - Migrated API from v0.6 to v0.7;
  - Removed the `UploadcareClient.storeGroup()` and `UploadcareGroup.store()` methods, as its
    endpoint has been removed;
  - Added event types of webhooks;
  - Removed the sorting methods by file size, as its parameter has been removed from the API query 
    parameters.
- Example:
  - Removed sorting options by file size from `UploadFragment`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 # History
 
+## 3.4.0
+- Library:
+  - Migrated API from v0.6 to v0.7
```
Comment (rsedykh) [inline CHANGELOG.md:7]:
```suggestion
  - Migrated REST API support from v0.6 to v0.7.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
 
 # History
 
+## 3.4.0
```
Comment (rsedykh) [inline CHANGELOG.md:5]:
```suggestion
## 4.0.0
```

I think this is kinda breaking enough change for a major version bump, don't you think?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+- Library:
+  - Migrated API from v0.6 to v0.7
+  - Removed the `UploadcareClient.storeGroup()` and `UploadcareGroup.store()` methods, as its
+    endpoint has been removed.
```
Comment (rsedykh) [inline CHANGELOG.md:9]:
```suggestion
  - Removed the `UploadcareClient.storeGroup()` and `UploadcareGroup.store()` methods, as their
     endpoints have been removed.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,17 @@
...
+  - Added event types of webhooks.
```
Comment (rsedykh) [inline CHANGELOG.md:10]:
```suggestion
  - Added support for new event types of webhooks.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,17 @@
...
+  - Removed the sorting methods by file size, as its parameter has been removed from the API query 
+    parameters.
```
Comment (rsedykh) [inline CHANGELOG.md:12]:
```suggestion
  - Removed sorting methods by file size, as the corresponding parameter has been removed from
    the API query parameters.
```

**Review by rsedykh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (onthecrow) [reply CHANGELOG.md:5]: Agree, but we should bump the version in gradle files as well.

Comment (rsedykh) [reply CHANGELOG.md:5]: Yep, I just didn't comment on every other line with a version. :-)

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,8 +5,8 @@
 ## 3.4.0
 - Library:
   - Migrated REST API support from v0.6 to v0.7.
-  - Removed the `UploadcareClient.storeGroup()` and `UploadcareGroup.store()` methods, as its
-    endpoint has been removed.
+  - Removed the `UploadcareClient.storeGroup()` and `UploadcareGroup.store()` methods, as their
+     endpoints have been removed.
```
Comment (rsedykh) [inline CHANGELOG.md:9]: Oops, one less space please. :-(

**Review by genme — APPROVED** [review]

**Review by rsedykh — APPROVED** [review]

**Pull request merged**